test/button_test.js

```diff
@@ -6,6 +6,7 @@ const clipboardy = require("clipboardy");
 const firefox = require("selenium-webdriver/firefox");
 
 const Context = firefox.Context;
+const MAX_TIMES_TO_SHOW = 5; // this must match MAX_TIMES_TO_SHOW in bootstrap.js
```

Comment:
Could you move `MAX_TIMES_TO_SHOW` to `Config.jsm`? That way both `bootstrap.js` and this file could import it and have to make sure the code stays in sync.